maintainers/scripts/luarocks-packages.csv

```diff
@@ -110,6 +110,7 @@ nvim-nio,,,,,,mrcjkb
 pathlib.nvim,,,,,,
 penlight,https://github.com/lunarmodules/Penlight.git,,,,,alerque
 plenary.nvim,https://github.com/nvim-lua/plenary.nvim.git,,,,5.1,
+psl,,,,0.3,,
```

Comment:
do you want to set yourself as maintainer ?

Reply:
No, not really. The library seems to be mostly dead, though.